packages/ag-charts-community/src/chart/chart.ts

```diff
@@ -1040,7 +1038,7 @@ export abstract class Chart extends Observable implements AgChartInstance {
         this.lastInteractionEvent = undefined;
     });
     protected handlePointer(event: InteractionEvent<'hover'>) {
-        const { lastPick, hoverRect } = this;
+        const { lastPick, seriesRect } = this;
```

Comment:
should this be using `seriesPaddedRect`? because `hoverRect` was growing from that rather than `seriesPaddedRect`

Reply:
Good spot, I've updated it to use the padded rect throughout instead.